src/ethereum_test_tools/common/types.py

```diff
@@ -1115,38 +1503,31 @@ def default(self, obj):
                 "gas": hex(obj.gas_limit),
                 "value": hex(obj.value),
                 "input": code_to_hex(obj.data),
-                "to": "0x" + int.to_bytes(obj.to, length=20, byteorder="big").hex()
-                if obj.to is int
-                else obj.to,
+                "to": "0x" + address_to_bytes(obj.to).hex() if obj.to is not None else None,
                 "accessList": obj.access_list,
                 "protected": obj.protected,
                 "secretKey": obj.secret_key,
                 "maxFeePerDataGas": hex_or_none(obj.max_fee_per_data_gas),
             }
 
             if obj.blob_versioned_hashes is not None:
-                hashes: List[str] = []
-                for h in obj.blob_versioned_hashes:
-                    if type(h) is str:
-                        hashes.append(h)
-                    elif type(h) is bytes:
-                        if len(h) != 32:
-                            raise TypeError("improper byte size for blob_versioned_hashes")
-                        hashes.append("0x" + h.hex())
-                    else:
-                        raise TypeError("improper type for blob_versioned_hashes")
-                tx["blobVersionedHashes"] = hashes
+                tx["blobVersionedHashes"] = [
+                    "0x" + hash_to_bytes(h).hex() for h in obj.blob_versioned_hashes
```

Comment:
We could also have a hash_to_hex(...) function in conversions.py, but I think we could do refactor for these in the future

```suggestion
                    hash_to_hex(h) for h in obj.blob_versioned_hashes
```

Reply:
Ah we have `hash_string`. We really need a better way to organize these.